### PR TITLE
color-schemes: remove sass $white

### PR DESCRIPTION
**Pull request by flootr**
#### Changes proposed in this Pull Request

* Remove remaining usages of `$white` and replace its occurrences with `--color-white`

#### Testing instructions

**code**:

- make sure each assignment is correct and there are no typos
- make sure there are no occurrences left of `$white` in Calypso code

**visually:**:

- smoke test Calypso and make sure none of the new gray usages look wrong

### CODE REVIEW

File: client/components/accordion/style.scss [hunk 1]
```diff
@@ -1,8 +1,5 @@
-
 $accordion-subtitle-height: 16px;
-$accordion-background-collapsed: $white; // same as body
```
Comment (blowery) [inline client/components/accordion/style.scss]: yay

File: client/extensions/woocommerce/components/table/style.scss [hunk 2]
```diff
@@ -87,7 +87,7 @@
 			overflow: hidden;
 
 			&::after {
-				background-image: linear-gradient( to right, rgba( 255, 255, 255, 0 ) 0%, $white 90% );
+				background-image: linear-gradient( to right, rgba( 255, 255, 255, 0 ) 0%, var( --color-white ) 90% );
```
Comment (blowery) [inline client/extensions/woocommerce/components/table/style.scss]:
```suggestion
				background-image: linear-gradient( to right, rgba( var( --color-white-rgb ), 0 ) 0%, var( --color-white ) 90% );
```

File: client/gutenberg/extensions/todo/editor.scss [hunk 3]
```diff
@@ -19,7 +19,7 @@ $light-gray-400: #e8eaeb;
 $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
-$white: #fff;
+var( --color-white ): #fff;
```
Comment (blowery) [inline client/gutenberg/extensions/todo/editor.scss]: this should just be dropped?

Comment (flootr) [reply client/gutenberg/extensions/todo/editor.scss]: yes, good catch

**Pull request merged**